.travis-ci.sh

```diff
@@ -13,7 +13,12 @@ install-bootstrap () {
     eval $(opam config env --root=$OPAMBSROOT)
     if [ "$OPAM_TEST" = "1" ]; then
         opam pin add opam-file-format "git://github.com/ocaml/opam-file-format.git" --no-action --yes
-        opam install ocamlfind lwt.3.0.0 cohttp.0.22.0 ssl cmdliner dose3 opam-file-format --yes
+        if [ "$OCAML_VERSION" != "4.01.0" ] ; then
+          EXTRAS="lwt.3.0.0 cohttp.0.22.0 ssl"
```

Comment:
No idea! I was just moving this out of the way as it doesn't build on 4.01.0 and neither does lib-ext (but this was a silly way of trying to do it, which I've just changed).